app/assets/javascripts/profiles.coffee

```diff
@@ -0,0 +1,34 @@
+updateDynamicFields = (el) ->
+  type = el.val()
+
+  if (type != "")
+    target = el.closest(".js-nested-fields").find(".js-item-content")
+
+    fields = target.children()
+    fields.hide()
+    fields.keyup ->
+      fields.val($(this).val())
+
+    switch (type)
+      when "text"
+        target.find("textarea").attr("placeholder", "Text body").show()
```

Comment:
These are chained jQuery functions. Firstly we find textarea, then we change the placeholder to "Text body" and after that, we show this element.

Reply:
Yes, you are right.

Why we can't specify `placeholder` in HTML?

Reply:
I have decided to do this that way, since in the future we might have multiple item types using the same textarea or input.

So for instance, we might have `video type` using input field just as `image type`. Thanks to setting the placeholder with JavaScript we are able to provide different one for each type.
When we select `image type` we will see a placeholder "Image URL", however `video type`, would display "Video URL".